src/controls.lua

```diff
@@ -98,13 +98,17 @@ function Controls.create(joystick)
 			playerMenu      = {joystick, "start"},
 			zoomOut		= {mouse, "-yWheel"},
 			zoomIn		= {mouse, "yWheel"},
-			cursorX 	= {joystick, 1, "change"},
-			cursorY 	= {joystick, 2, "change"},
+			cursor          = {
+				device  = joystick,
+				offset  = "change",
+				x       = 1,
+				y       = 2
+			},
```

Comment:
I changed the structure of cursors in the key bindings so that it's easier to tell whether something is a mouse or joystick.